site/visitor-console/vite.config.ts

```diff
@@ -5,4 +5,7 @@ import eslintPlugin from "vite-plugin-eslint";
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [eslintPlugin({}), react()],
+  define: {
+    global: {},
+  },
```

Comment:
This is causing `npm run build` to fail. If I comment it out then the build succeeds again, but the page doesn't render anything if I load it via `npm run start`.

Reply:
Weird behavior, I forgot to do the npm run build step before making a pr. Maybe put the define global in the readme for local testing?

Reply:
Can you try https://github.com/vitejs/vite/discussions/5912#discussioncomment-5569850 instead? It worked on my machine for both `npm run start` and `npm run build`.